.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -148,16 +145,38 @@ jobs:
           username: ${{ steps.secrets.outputs.REGISTRY_HUB_DOCKER_COM_USR }}
           password: ${{ steps.secrets.outputs.REGISTRY_HUB_DOCKER_COM_PSW }}
 
+      - id: "auth"
+        name: "Authenticate to Google Cloud"
+        uses: "google-github-actions/auth@v2"
+        with:
+          token_format: "access_token"
+          workload_identity_provider: "projects/271764561088/locations/global/workloadIdentityPools/github/providers/camunda"
+          service_account: "github-actions-camunda@team-infosec.iam.gserviceaccount.com"
+
+      - name: Login to infosec registry
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3
+        with:
+          registry: europe-west1-docker.pkg.dev
+          username: oauth2accesstoken
+          password: ${{ steps.auth.outputs.access_token }}
+
       - name: Checkout release branch
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
         with:
           ref: ${{ env.BRANCH }}
           fetch-depth: 0
 
+
+      - name: "Read Java / Version Info"
+        id: "pom-info"
+        uses: YunaBraska/java-info-action@main
+        with:
+          work-dir: ./optimize
+
       - name: Configure GitHub user
         run: |
-          git config --global user.name "optimize-release"
-          git config --global user.email "ci@optimize.camunda.cloud"
+          git config --global user.email "github-actions[release]"
+          git config --global user.name "github-actions[release]@users.noreply.github.com"
```

Comment:
is it expected that the username looks like an email and the email looks like a username?

Reply:
I don't know, this is a chrrey-picked commit from someone else. Since t[his is how it is on main now](https://github.com/camunda/camunda/blob/bfdecce3149ceade4bd201b10942d291a4cccd47/.github/workflows/optimize-release-optimize-c8-only.yml#L177) I am assuming this is how it's supposed to be

Reply:
but you're absolutely right, this looks very suspicious. I will correct this. Well spotted. We need to move this to main as well

Reply:
@buccarel could you please create a PR to apply this correction in the main branch as well?